test-app/build-tools/static-binding-generator/src/main/java/org/nativescript/staticbindinggenerator/generating/parsing/methods/impl/JavaClassUtils.java

```diff
@@ -0,0 +1,65 @@
+package org.nativescript.staticbindinggenerator.generating.parsing.methods.impl;
+
+import org.apache.bcel.classfile.JavaClass;
+import org.apache.bcel.classfile.Method;
+import org.nativescript.staticbindinggenerator.generating.parsing.classes.hierarchy.generics.GenericHierarchyView;
+import org.nativescript.staticbindinggenerator.generating.parsing.methods.JavaMethod;
+import org.nativescript.staticbindinggenerator.generating.parsing.methods.ReifiedJavaMethod;
+import org.nativescript.staticbindinggenerator.naming.BcelNamingUtil;
+
+import java.util.Map;
+
+class JavaClassUtils {
+    private static final String BASE_CLASS_NAME = "java.lang.Object";
```

Comment:
what about moving this one somewhere and use it everywhere instead of declaring it several times